### ISSUE TITLE
anchor the first top solid layer

### ISSUE THREAD

**Issue by supermerill** [supermerill/Slic3r#140]
like https://3dprintingindustry.com/wp-content/uploads/2019/02/pathio-vs-old-slicers-768x1099.jpg
Add a setting to activate
if more than 1/2 ? perimeter(s) at this position, if this perimeter intersect with a solid bridged layer, then don't print this perimeter

**Comment by JS3910** [comment]
That looks good, but why in this illustration are only the layers at the middle height inset into the perimeters? Should it not also apply to the topmost one?

**Comment by supermerill** [comment]
i think it should.

**Comment by JS3910** [comment]
Yep, I don't understand any conceptual difference between the first top layer on the two "side" parts vs the topmost bit, but that's on the original source for the image of course. I'm assuming that these are implied to be bridges going into the plane of the screen or have support not shown in the image, or this would not work so well at all :D.
